editor/svg-editor.css

```diff
@@ -1246,10 +1246,6 @@ ul li.current {
   margin-left: 0;
 }
 
-#tool_prefs_option {
-  float: right;
-}
-
```

Comment:
I see this selector is still used in one of the cypress tests (`ui-test-helper.js`). Please adjust that test accordingly when you have a chance. Otherwise, from what I can tell without having tested locally yet, the changes looks fine.

Reply:
I updated the selector to '#tool_editor_prefs' in `ui-test-helper.js`

Reply:
Cool. Just make sure that test still passes. We want to at least be as good as before.

Reply:
All tests are ok except 4 accessibility violations

Reply:
Ok, great, thanks!